acp/identity/identity.go

```diff
@@ -158,3 +158,12 @@ func didFromPublicKey(publicKey *secp256k1.PublicKey, producer didProducer) (str
 	}
 	return did.String(), nil
 }
+
+// IntoRawIdentity converts an `Identity` into a `RawIdentity`.
+func (identity Identity) IntoRawIdentity() RawIdentity {
+	return RawIdentity{
+		PrivateKey: hex.EncodeToString(identity.PrivateKey.Serialize()),
+		PublicKey:  hex.EncodeToString(identity.PublicKey.SerializeCompressed()),
+		DID:        identity.DID,
+	}
+}
```

Comment:
todo: Since you have made a function for this, why not call this at the end of `Generate()` in the file: `acp/identity/generate.go` right at the `return` stage.

suggestion: Also perhaps can move this function and the `RawIdentity` struct to a new file like `raw_identity.go` perhaps?

Reply:
extracted common code.

`IntoRawIdentity` belongs to `Identity`, so I think it should stay there. 

I think we will need in the near future `IntoIdentity` for `RawIdentity`. This one will to into `raw_identity.go`